addOns/ascanrulesBeta/src/main/java/org/zaproxy/zap/extension/ascanrulesBeta/HttpParameterPollutionScanRule.java

```diff
@@ -104,15 +105,10 @@ public void scan() {
                     try {
                         sendAndReceive(newMsg);
                     } catch (IllegalStateException | UnknownHostException ex) {
-                        if (log.isDebugEnabled())
-                            log.debug(
-                                    "Caught "
-                                            + ex.getClass().getName()
-                                            + " "
-                                            + ex.getMessage()
-                                            + " when accessing: "
-                                            + newMsg.getRequestHeader().getURI().toString()
-                                            + "\n The target may have replied with a poorly formed redirect due to our input.");
+                        log.debug(
+                                "Caught {} {} when accessing: {}.\n The target may have replied with a poorly formed redirect due to our input.",
+                                ex.getClass().getName().concat(ex.getMessage()),
```

Comment:
Was the `concat`change done on purpose?

Reply:
Nope, I have no idea how I managed to do that 👎